### PR TITLE
[Docs] update custom meshkit code usage in meshery-server docs

### PR DESCRIPTION
**Pull request by alloydm**
**Notes for Reviewers**

This PR adds #
The steps to use custom meshkit code for meshery-server development
Screenshot 2024-07-30 at 6 07 45 PM

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit eefd8383e98d1539d617d5e5f56639ad23779d65 at: https://66a8dfca1188420c9f23444d--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/meshery-server.md [hunk 1]
```diff
@@ -82,3 +82,26 @@ The server log levels can be configured at runtime by changing the env variable
...
+### Using custom Meshkit code for Meshery server development
+
+<ol>
+  <li>
+    <p>Checkout <strong>meshery</strong> and <strong>meshkit</strong> repository in adjacent directories.</p>
+    {% capture code_content %}
+mkdir meshery
+cd meshery
+git clone https://github.com/meshery/meshery.git
+git clone https://github.com/meshery/meshkit.git
```
Comment (leecalcote) [inline docs/pages/project/contributing/meshery-server.md:95]:
```suggestion
$ git clone https://github.com/meshery/meshery.git
$ git clone https://github.com/meshery/meshkit.git
```

Removing explicit checkout folder name.

File: docs/pages/project/contributing/meshery-server.md [hunk 2]
```diff
@@ -82,3 +82,24 @@ The server log levels can be configured at runtime by changing the env variable
...
+    {% endcapture %}
+    {% include code.html code=code_content %}
+  </li>
+  <li>
+    <p>In your <strong>meshery</strong> go.mod, update the meshkit package.</p>
+    {% capture code_content %}
+github.com/layer5io/meshkit => ../meshkit
+    {% endcapture %}
+    {% include code.html code=code_content %}
+  </li>
```
Comment (leecalcote) [inline docs/pages/project/contributing/meshery-server.md:103]:
```suggestion
  <li>
    <p>In your <code>meshery</code> go.mod, update the meshkit package.</p>
    {% capture code_content %}
github.com/layer5io/meshkit => ../meshkit
    {% endcapture %}
    {% include code.html code=code_content %}
    <p>Remember to remove this go.mod change when creating pull requests.</p>
  </li>
```

**Review by leecalcote — APPROVED** [review]
Good work, @alloydm :thu

**Pull request merged**